content/en/service_management/events/correlation/patterns.md

```diff
@@ -27,7 +40,7 @@ To create or adjust a pattern-based correlation:
    : is any event that is available in your Event Explorer as an event, but does not have an associated monitor or alert.
    
    Add grouping tags
-   : in addition to grouping by out-of-the-box tags like `Service` or `Env`, you can also group by a custom tag.
+   : to add new grouping tags, this is same as adding new event facet. 
```

Comment:
Can we link to the docs on new event facets? Is this the correct [page](https://docs-staging.datadoghq.com/wendy.jiang/update-event-management-doc/service_management/events/explorer/facets/#create-a-facet)? 

```suggestion
   : to add new grouping tags, this is same as adding new event facet. 
```